tests/v2/validation/provisioning/hosted/aks/scaling/scaling_nodepools.go

```diff
@@ -0,0 +1,102 @@
+package scaling
+
+import (
+	"time"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/cloudcredentials"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters/aks"
+	np "github.com/rancher/rancher/tests/framework/extensions/clusters/aks/resources"
+	"github.com/sirupsen/logrus"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+)
+
+// ScalingAKSNodePools is a helper function that tests scaling of an AKS node pool by adding a new one and then deleting it.
+func ScalingAKSNodePools(client *rancher.Client, oldCluster *management.Cluster, displayName string, cloudCredential *cloudcredentials.CloudCredential) (*management.Cluster, error) {
+	logrus.Infof("======================================")
+	logrus.Infof("TEST CASE: Scaling AKS Node Pools")
+	logrus.Infof("======================================")
```

Comment:
No a fan of these log statements, as this isn't a test case more of an action

Reply:
Want me to just remove it altogether? My original thought behind was to make it a bit easier to distinguish between test cases. But I guess since now we have it split into different test cases, doesn't really matter too much now huh?

Reply:
I removed it from previous comment I made. Doesn't really need to be there now that we refactored the test functions anyway.

Reply:
Yes totally agree they don't need to be there at all